docs/_writing-tests/general-guidelines.md

```diff
@@ -112,6 +112,30 @@ either installed or to have specific metrics. As such, in most cases
 when a known font is needed Ahem should be used. In other cases,
 `@font-face` should be used.
 
+#### Be Conservative
+
+Sometimes, when authoring tests for one web platform feature, it may be
+convenient to use some unrelated feature (e.g. web browser APIs or ECMAScript
```

Comment:
That phrase only describes features that would *not* be acceptable according to
the criteria we've identified here. This text needs to apply to *any* feature
since the explanation that follows describes how to distinguish between
"acceptable" and "unacceptable." I chose the words "unrelated" and "extraneous"
to differentiate the features under test (whose validity is unquestionable)
from those that are there to facilitate the test (whose validity is determined
by the criteria documented here).

To try make this more clear, I've rewritten that sentence:

> it may be convenient to use some other feature (e.g. a web browser API or
> ECMAScript language feature) that is not directly related to the test itself